web/components/ui/followers/SingleFollower/SingleFollower.tsx

```diff
@@ -19,7 +19,9 @@ export const SingleFollower: FC<SingleFollowerProps> = ({ follower }) => (
         </Col>
         <Col span={18}>
           <Row className={styles.name}>
-            <Typography.Text ellipsis>{follower.name}</Typography.Text>
+            {/* using ternary operrator for follower name,
```

Comment:
The double pipe (`||`) isn't the ternary operator, it's the [logical OR operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR). But it's doing the right thing! The comment isn't necessary so you can probably just remove it. See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) for

Reply:
Sorry, I mistakenly write ternary operator instead of OR operator, sure I'll make the suggested changes.